mu4/importexport/internal/braille/exportbraille.cpp

```diff
@@ -1683,6 +1683,11 @@ QString ExportBraille::brailleBarline(BarLine* barline)
     case BarLineType::END_START_REPEAT:
         barlineBraille = BRAILLE_BARLINE_START_REPEAT + " " + BRAILLE_BARLINE_END_REPEAT;
         break;
+    case BarLineType::REVERSE_END:
+    //case BarLineType::REVERSE_FINALE: // duplicate of the above, alternative name
+    case BarLineType::HEAVY:
+    case BarLineType::DOUBLE_HEAVY:
+         break; // ToDo ?!
```

Comment:
@andreituicu how to deal with those? They got added to MuseScore recently

Reply:
Actually I don't think we should be ignoring them, at least report them as normal barlines?